Resources/Prototypes/Atmospherics/gases.yml

```diff
@@ -5,6 +5,7 @@
   heatCapacityRatio: 1.4
   molarMass: 32
   color: 2887E8
+  reagent: Oxygen
 
 - type: gas
```

Comment:
CO2 needs a chem equivalent for CO2 poisoning. Nitrogen needs a chem equivalent. Water Vapor should become water (and maybe contribute to suffocation/warning messages in high concentrations).

Reply:
does nitrogen need a chem equivalent? for what
co2 & water vapor sure

Reply:
Nitrogen is utilized by Vox.

Reply:
On making more gasses reagents, that probably requires changing how exhaling gasses works.
Currently, you need to add a `CreateLungGas` metabolism to turn the reagent back into gas, which seems weird. E.g., for humans you would need to:
- inhale nitrogen gas
- turned gas into reagents and add reagents to solution
- remove gas
- reagent gets metabolised, adding the same gas back to lungs
- gas is exhaled

At bare minimum, there should probably just be a `ReagentToGas()` function that does the opposite of `GasToReagent()`, so you don't need to add a bunch of metabolisms. But an even better solution would probably be to just not turn gasses into reagents if the lung has no associated metabolism for that reagent. Maybe both, in case a reagent is not fully metabolised by lungs?

Reply:
yeah i thought about this a little more and I think i'll have to add a field on the lung component for which gases should be processed into reagents at all to avoid that